packages/api/src/utils/db-client.js

```diff
@@ -125,20 +125,20 @@ export class DBClient {
         updated_at: now,
       })
       .select(this.uploadQuery)
-      .eq('id', data.id)
+      .eq('content_cid', data.content_cid)
       .eq('user_id', data.user_id)
       .is('deleted_at', null)
       .single()
 
     if (status === 406) {
-      throw new Error(`Status 406, cannot update ${data.id}`)
+      throw new Error(`Status 406, cannot update ${data.content_cid}`)
     }
 
     if (!upload) {
       throw new Error(
-        `Cannot update upload ${JSON.stringify(data.id)} ${JSON.stringify(
-          status
-        )}`
+        `Cannot update upload ${JSON.stringify(
+          data.content_cid
+        )} ${JSON.stringify(status)}`
```

Comment:
Why are these strings/numbers being passed to `JSON.stringify`?

Reply:
Leftover cleanup, good catch!